CMakeLists.txt

```diff
@@ -695,7 +676,6 @@ if (${HDF_CFG_NAME} MATCHES "Debug" OR ${HDF_CFG_NAME} MATCHES "Developer")
   if (HDF5_ENABLE_INSTRUMENT)
     set (H5_HAVE_INSTRUMENTED_LIBRARY 1)
   endif ()
-  mark_as_advanced (HDF5_ENABLE_INSTRUMENT)
```

Comment:
Why the removal of this?

Reply:
What gets made advanced is fairly arbitrary so it's not like this is a major change, but since the dependent part is based on `HDF5_ENABLE_TRACE` it has a similar effect of `mark_as_advanced` in only some cases. No big deal though